src/common.browser/MicAudioSource.ts

```diff
@@ -64,7 +64,9 @@ export class MicAudioSource implements IAudioSource {
     public constructor(
         private readonly privRecorder: IRecorder,
         private readonly deviceId?: string,
-        audioSourceId?: string) {
+        audioSourceId?: string,
+        private readonly mediaStream?: MediaStream
```

Comment:
>mediaStream [](start = 25, length = 11)

Why create another member variable instead of moving the initialization of the existing privMediaStream?

Reply:
Yes, makes sense. I fixed it.